llm_on_ray/finetune/finetune.py

```diff
@@ -193,92 +242,43 @@ def train_func(config: Dict[str, Any]):
         }
     )
 
-    optimizer = common.optimizer.Optimizer.registory.get("DefaultOptimizer")()(
-        model,
-        config={
-            "name": config["Training"]["optimizer"],
-            "config": {"lr": config["Training"]["learning_rate"]},
-        },
```

Comment:
Why remove optimizer?

Reply:
optimizer will create in `transformers.Trainer` or `optimum.habana.transformers.GaudiTrainer`